.github/workflows/test_nixos_minimal.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         with:
           image: asterinas/asterinas:0.16.1-20251130
           options: --privileged -v /dev:/dev -v ${{ github.workspace }}:/root/asterinas
-          run: make run_nixos NIXOS_DISK_SIZE_IN_MB=4096 NIXOS_STAGE_2_INIT='/bin/sh -l -c "hello-asterinas"' || true
+          run: make run_nixos NIXOS_DISK_SIZE_IN_MB=4096 NIXOS_RUN_TEST=true || true
```

Comment:
`NIXOS_RUN_TEST` and `NIXOS_TEST_COMMAND` can be combined into one. If `NIXOS_TEST_COMMAND` is non-empty, then it implies running a test.